org-roam.el

```diff
@@ -1707,26 +1709,33 @@ Return added alias."
     alias))
 
 ;;;###autoload
-(defun org-roam-alias-delete ()
-  "Delete an alias from Org-roam file."
+(defun org-roam-alias-delete (&optional alias)
+  "Delete an ALIAS from Org-roam file.
+
+When ALIAS is nil, it is read interactively from the list of
+existing aliases."
   (interactive)
   (unless org-roam-mode (org-roam-mode))
   (if-let ((aliases (org-roam--extract-titles-alias)))
-      (let ((alias (completing-read "Alias: " aliases nil 'require-match)))
+      (let ((alias (or alias
+                       (completing-read "Alias: " aliases nil 'require-match))))
         (org-roam--set-global-prop
          "roam_alias"
          (combine-and-quote-strings (delete alias aliases)))
         (org-roam-db--update-file (buffer-file-name (buffer-base-buffer))))
-    (user-error "No aliases to delete")))
+    ;; error only during interactive usage
+    (unless alias
```

Comment:
I've added `user-error` here just to give feedback to the interactive users of this function when there is nothing to delete. I think for non-interactive usages it's better to simply noop. Same for tags.